mycroft/skills/roku_control/__init__.py

```diff
@@ -0,0 +1,126 @@
+        self.roku = None
+        if self.ip:
+            self.roku = Roku(self.ip)
+
+    def initialize(self):
+        self.load_data_files(dirname(__file__))
+        self.register_apps()
+
+        roku_launch_intent = IntentBuilder("RokuLaunchIntent")\
+            .require("RokuKeyword")\
+            .require("RokuLaunchKeyword")\
+            .require("Application")\
+            .build()
+        self.register_intent(roku_launch_intent,
+                             self.handle_roku_launch_intent)
+
+        roku_home_intent = IntentBuilder("RokuHomeIntent")\
+            .require("RokuKeyword")\
+            .require("RokuHomeKeyword")\
+            .build()
+        self.register_intent(roku_home_intent,
+                             self.handle_roku_home_intent)
+
+        roku_apps_intent = IntentBuilder("RokuAppsIntent")\
+            .require("RokuKeyword")\
+            .require("RokuAppKeyword")\
+            .build()
+        self.register_intent(roku_apps_intent,
+                             self.handle_roku_apps_intent)
+
+        roku_play_intent = IntentBuilder("RokuPlayIntent")\
+            .require("RokuKeyword")\
+            .require("RokuPlayKeyword")\
+            .build()
+        self.register_intent(roku_play_intent,
+                             self.handle_roku_play_intent)
+
+        roku_pause_intent = IntentBuilder("RokuPauseIntent")\
+            .require("RokuKeyword")\
+            .require("RokuPauseKeyword")\
+            .build()
+        self.register_intent(roku_pause_intent,
+                             self.handle_roku_pause_intent)
+
+    def register_apps(self):
+        if self.roku:
+            for app in self.roku.apps:
+                self.register_vocabulary(app.name, "Application")
+
+    def handle_roku_launch_intent(self, message):
+        if self.roku:
+            app = message.metadata.get('Application')
+            self.speak_dialog("launch.app", {"Application": app})
+            self.roku[app].launch()
+        else:
+            self.speak_dialog("no.device")
+
+    def handle_roku_home_intent(self, message):
+        if self.roku:
+            self.speak_dialog("home")
+            self.roku.home()
+        else:
+            self.speak_dialog("no.device")
+
+    def handle_roku_apps_intent(self, message):
+        if self.roku:
+            self.speak_dialog("apps")
+            for app in self.roku.apps:
+                self.speak(app.name)
+        else:
+            self.speak_dialog("no.device")
+
+    def handle_roku_play_intent(self, message):
+        if self.roku:
+            self.speak_dialog("play")
+            self.roku.play()
+        else:
+            self.speak_dialog("no.device")
+
+    def handle_roku_pause_intent(self, message):
+        if self.roku:
+            self.speak_dialog("pause")
+            self.roku.play()
+        else:
+            self.speak_dialog("no.device")
+
+    def stop(self):
+        pass
```

Comment:
So to be clear, would this apply when someone says `stop roku`, or when someone says `stop` when the roku skill is doing something? If it's the first, that should just be as simple as adding another `self.roku.play` call, as that will pause something that is currently playing.